app/views/citizens/contact_providers/show.html.erb

```diff
@@ -1,6 +1,8 @@
-<div class="interruption-panel">
-  <%= page_template(template: :basic) do %>
-    <h1 class="govuk-heading-l"><%= t(".title") %></h1>
-    <p><%= t(".they_will_check") %></p>
-  <% end %>
+<div class="moj-interruption-card">
+  <div class="moj-interruption-card__content">
+    <%= page_template(template: :basic) do %>
+      <h1 class="govuk-heading-l moj-interruption-card__heading"><%= t(".title") %></h1>
+      <p class="govuk-body moj-interruption-card__body"><%= t(".they_will_check") %></p>
```

Comment:
Component markup seems incorrect, should it be 

```
    <div class="govuk-body-l moj-interruption-card__body">
        <p><%= t(".they_will_check") %></p>
    </div>
```

Reply:
@naseberry Is the only change the `govuk-body-l`? Sara suggested doing the following:

> - Headings should be large size
> - Body text should be medium size
> - Content width should be 2/3s
> - Headings should be one sentence, with subsequent sentences written as body text

Is there something else I'm not spotting?

Reply:
Then, could we add the design suggestions to the commit body for future reference?

Reply:
Sure. Should I put it in every commit message? It's in quite a few places